src/solver/conv_mlir_igemm_wrw_xdlops.cpp

```diff
@@ -105,6 +105,7 @@ ConvSolution ConvMlirIgemmWrWXdlops::GetSolution(const ConvolutionContext& ctx,
 
     result.invoker_factory = conv::MakeMlirWrWInvokerFactory(ctx);
     result.construction_params.push_back(construction_parameters);
+    result.workspace_sz = GetWorkspaceSize(ctx);
```

Comment:
Pass required workspace size to `MakeMlirWrWInvokerFactory()`

Reply:
Will do. Do we still need to set `result.workspace_sz` here? And where inside `MakeMlirWrWInvokerFactor` should I keep track of the required workspace size?

Reply:
@atamazov I have submitted a patch. Please help review it again. Thanks. I'm still keeping `result.workspace_sz` be set here.

Reply:
>  Do we still need to set result.workspace_sz here? 

yes.